code/modules/antagonists/ert/ert.dm

```diff
@@ -351,3 +351,23 @@
 /datum/antagonist/ert/families/beatcop/military/New()
 	. = ..()
 	name_source = GLOB.commando_names
+
+/datum/antagonist/ert/marine
+	name = "Marine Commander"
+	outfit = /datum/outfit/centcom/ert/marine
+	role = "Commander"
+
+/datum/antagonist/ert/marine/security
+	name = "Marine Heavy"
+	outfit = /datum/outfit/centcom/ert/marine/security
+	role = "Security Officer"
```

Comment:
he aint a security officer doe

Reply:
you see this is copied from the ert sec officer and i didn't read what it actually does 😎